app/knight_info/potion.py

```diff
@@ -0,0 +1,13 @@
+class KnightPotion:
```

Comment:
It's ok, but I would say not necessary to add Knight prefix to the classes, since it's a small project -- take it into consideration!

Reply:
Done